server/api/router.go

```diff
@@ -101,6 +101,7 @@ func createRouter(ctx context.Context, prefix string, svr *server.Server) (*mux.
 	clusterRouter.HandleFunc("/store/{id}", storeHandler.Delete).Methods("DELETE")
 	clusterRouter.HandleFunc("/store/{id}/state", storeHandler.SetState).Methods("POST")
 	clusterRouter.HandleFunc("/store/{id}/label", storeHandler.SetLabels).Methods("POST")
+	clusterRouter.HandleFunc("/store/{id}/label/force", storeHandler.SetLabelsForce).Methods("POST")
```

Comment:
I prefer using URI parameter, such as `store/{id}/label?force`.

Reply:
OK, I will try it.